flixel/addons/display/FlxZoomCamera.hx

```diff
@@ -29,7 +29,7 @@ class FlxZoomCamera extends FlxCamera
 	 */
 	private var _zoomMargin:Float;
 	
-	public function new(X:Int, Y:Int, Width:Int, Height:Int, Zoom:Float = 0)
+	public function new(X:Int, Y:Int, Width:Int, Height:Int, Zoom:Float = 0, ZoomSpeed:Float = 25, ZoomMargin:Float = 0.25)
```

Comment:
I feel like these don't necessarily need to be in the constructor, having a lot of arguments always has an impact on readability.

Plus, your forgot to actually use the parameter values to initialize the variables. ;)